zeppelin-client-examples/src/main/java/org/apache/zeppelin/client/examples/ZeppelinClientExample.java

```diff
@@ -56,6 +56,8 @@ public static void main(String[] args) throws Exception {
               "%python\nimport time\ntime.sleep(5)\nprint('done')");
       zClient.submitParagraph(noteId, paragraphId2);
       zClient.waitUtilParagraphRunning(noteId, paragraphId2);
+      // It's also ok here to call zClient.cancelNote(noteId);
```

Comment:
Need to add another test case for `cancelNote` instead of just leaving a comment